pkg/domain/infra/abi/containers.go

```diff
@@ -1589,7 +1589,14 @@ func (ic *ContainerEngine) ContainerClone(ctx context.Context, ctrCloneOpts enti
 
 	if ctrCloneOpts.Destroy {
 		var time *uint
-		err := ic.Libpod.RemoveContainer(context.Background(), c, false, false, time)
+		state, err := c.State()
+		if err != nil {
+			return nil, err
+		}
+		if state == define.ContainerStateRunning && !ctrCloneOpts.Force {
```

Comment:
This log would not be shown on the remote client.  However, we can do an error check below:
```golang
if err != nil {
    if errors.Is(,err, define.ErrCtrStateInvalid) {
         // wrap the error error
    }
....
```
}